cvexporter/quay_images.py

```diff
@@ -0,0 +1,151 @@
+import os
+import requests
+import logging
+from functools import lru_cache as lc
+
+
+QUAY_API = str(os.getenv("QUAY_API", "https://quay.io/api/v1/"))
+QUAY_TOKEN = os.getenv("QUAY_TOKEN")
+
+
+def severities():
```

Comment:
I would recommend to change this to a constant SEVERITIES

Reply:
changed